internal/sidekick/internal/gcloud/command_test.go

```diff
@@ -49,6 +49,9 @@ func TestCommandYAML(t *testing.T) {
 			if err != nil {
 				t.Fatalf("failed to read file: %v", err)
 			}
+			stringData := strings.ReplaceAll(string(data), "\r\n", "\n")
```

Comment:
maybe:

```suggestion
			data = bytes.ReplaceAll(string(data), []byte("\r\n"), []byte("\n"))
```

Does that work?